Tarot Reader/tarot_card_reader.py

```diff
@@ -0,0 +1,101 @@
+import random
+
+
+# get tarot deck of 78 cards and 'predictions' from tarot.txt
+file_handle = open("tarot.txt", "r")
```

Comment:
You have not closed this file. Use `with open() as ` to read the file which doesn't require explicit file closing. Also , do this change in file path:
```suggestion
file_handle = open("./Tarot Reader/tarot.txt", "r")
```